src/docs/Components/Pagination/accessibilty.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Accessibility"
+order: 3
+---
+
+## Accessibility
+
+In order to guarantee the accessibility of the component, please make sure to respect the following rules when integrating the component into your pages:
```

Comment:
In order to guarantee the accessibility of the component => In order to guarantee the component accessibility

Reply:
Fix